standard/expressions.md

```diff
@@ -4097,7 +4122,7 @@ Note that some conversions, such as user defined conversions, are not possible w
 >     }
 >
 >     public T G<T>(object o)
->         where T : Attribute
+>         where T : System.Attribute
```

Comment:
Do we need this change, if standalone-lib is going to have using directives?

Reply:
reversed.